tasks/main.yml

```diff
@@ -27,6 +27,23 @@
   when: docker_daemon_options.keys() | length > 0
   notify: restart docker
 
+- name: Ensure systemd drop-in directory exists
+  file:
+    path: /etc/systemd/system/docker.service.d
```

Comment:
This hard coded path could be a problem on systems with custom configurations. 
It should be changeable like "{{ DOCKER_SERVICE_D_PATH|default("/etc/systemd/system/docker.service.d") }}".